lib/Backend/GlobOpt.cpp

```diff
@@ -527,6 +532,9 @@ GlobOpt::ForwardPass()
     // Make sure we free most of them.
     Assert(freedCount >= spilledCount);
 
+    JitAdelete(alloc, this->changedSymsAfterIncBailoutCandidate);
```

Comment:
It goes away in GlobOpt::Optimize(), right after the all to ForwardPass().

---
In reply to: [72353163](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72353163) [](ancestors = 72353163,72335803)